package exceptionTest;

public class Test5 extends RuntimeException
{

	
	public static void main(String[] args) {
		
		throw new Test5();
	}

}
